pkgs/development/ocaml-modules/owl/default.nix

```diff
@@ -26,7 +27,6 @@ buildDunePackage rec {
     npy
   ];
 
-  doCheck = false;
+  doCheck = stdenv.hostPlatform.isLinux;
   # Tests fail with Clang: https://github.com/owlbarn/owl/issues/462
```

Comment:
This issue was also closed, though it is not clear to me if it was resolved.

Reply:
And if it really is about clang the condition should reflect that. You can override GCC vs clang, it's just different defaults on Linux/Darwin